extensions/identity-hub-client/build.gradle.kts

```diff
@@ -41,6 +41,7 @@ tasks.withType(org.openapitools.generator.gradle.plugin.tasks.GenerateTask::clas
 // Ensure compileJava depends on openApiGenerate
 val compileJava: JavaCompile by tasks
 val openApiGenerate: org.openapitools.generator.gradle.plugin.tasks.GenerateTask by tasks
```

Comment:
@ouphi This module still generates an HTTP client, that is not used (we use a custom one). We should remove the code generation from the build file.